CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Kanister
+
+## How Can I Contribute?
+
+### Reporting Bugs
+Before creating bug reports, please check the list of issues as you might find out that a similar issue is already created and you don't need to create one. Also when creating a bug report, please include as many details as possible as this information may help us resolve the issue faster.
+
+### Suggesting Enhancements
+You can suggest minor improvements to existing functionality and or a completely new feature. For that, we recommend creating an issue at https://github.com/kanisterio/kanister/issues or reaching out to us at our [Slack](https://join.slack.com/t/kanisterio/shared_invite/enQtNzg2MDc4NzA0ODY4LTU1NDU2NDZhYjk3YmE5MWNlZWMwYzk1NjNjOGQ3NjAyMjcxMTIyNTE1YzZlMzgwYmIwNWFkNjU0NGFlMzNjNTk) workspace.
+
+### Your first code contribution
+You can start by looking through `good-first-issue` issues. 
+  
+#### Local Developement
+Once we are done with the changes, we need to make sure, that the changes are not breaking anything and are passing the unit as well as the integration test.
+
+```bash
+#Run golint command to make sure your code is properly formatted
+make golint
+
+#build the project
+make build
```

Comment:
what does this do? Where does it generate the binaries?

Reply:
and it generates the binaries in `$GOPATH/bin` directory

Reply:
Resolving this conversation.